CHANGELOG.md

```diff
@@ -2,14 +2,14 @@
 ------------------------------------------------------------------------
 - Feature: [#2] The window limit has been increased from 12 to 64.
 - Feature: [#1652] Display the production of each industry from the previous month in the industry list.
+- Change: [#1014] Loading and saving save files and scenarios uses progress bars again.
```

Comment:
Isn't this a fix? I don't think we meant to break the progress bar.

Reply:
There's a fix listed as well, further down. But our implementations for loading and saving didn't have any progress bars in the first place.